subliminal/subtitle.py

```diff
@@ -276,13 +275,16 @@ def guess_matches(video, guess, partial=False):
     matches = set()
     if isinstance(video, Episode):
         # series
-        if video.series and 'series' in guess and sanitized_string_equal(guess['series'], video.series):
+        if video.series and 'title' in guess and sanitized_string_equal(guess['title'], video.series):
             matches.add('series')
+        # series
+        if video.title and 'episode_title' in guess and sanitized_string_equal(guess['episode_title'], video.title):
+            matches.add('title')
```

Comment:
guessit 2 will find one of them as `alternative_title`. Search for alternative_title in this file, and you will have some examples: https://github.com/guessit-io/guessit/blob/master/guessit/test/movies.yml

Keep in mind than guessit is not a "human", and it's just guessing :) So it's actually impossible to find out that a particular string is an alternative title, a movies series title (like James Bond), or even a productor or main actor of the movie ... It will just go into `alternative_title` list.